src/main/java/io/github/jhipster/online/domain/Jdl.java

```diff
@@ -80,16 +81,36 @@ public void setJdlMetadata(JdlMetadata jdlMetadata) {
 
     @Override
     public boolean equals(Object o) {
-        if (this == o) {
+        if (isSameObject(o)) {
```

Comment:
These are unnecessary changes IMO, just makes code more verbose and the methods are not even reused. Same for other methods added. Please revert back to original `equals` method